docs/guides/draftjs-blocks-entityrange-entitmap.md

```diff
@@ -0,0 +1,196 @@
+
+### DraftJS block, entityRanges and entityMap
+
+A quick side note on how the DraftJS block, entityRanges and entityMap works, in the context of the TranscriptEditor component. For the [adapters](./adapters.md) guide.
+
+
+#### Data Block
+
+TL;DR: a block is a representation of a paragraph (as an Immutable Record) in draftJs and you can have some custom data associated to it.
+
+But see the docs notes on [draftjs basics](https://github.com/bbc/react-transcript-editor/blob/master/docs/notes/draftjs/2018-10-01-draftjs-1-basics.md) to better understand the role of content block within the editor. As well as the draftJs official docs.
+
+Here's an example of a block, you can see it can contain some custom data, eg speaker name, list of words, and start time (which would be the start time of the first word).
+
+```js
+[
+  {
+    "text": "There is a day.", // text 
+    "type": "paragraph", // type of block 
+    "data": { //optional custom data
+      "speaker": "TBC 0",
+      "words": [
+       ...
+      ],
+      "start": 13.02
+    },
+    "entityRanges": [ // <-- entity ranges
```

Comment:
Don't think you really need this + line 34 as well.

Reply:
this? as in the custom data? or the example code?